spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -3490,3 +3437,6 @@ trait CometExpressionSerde {
       inputs: Seq[Attribute],
       binding: Boolean): Option[ExprOuterClass.Expr]
 }
+
+/** Marker trait for an expression that is not guaranteed to be 100% compatible with Spark */
+trait IncompatExpr {}
```

Comment:
Later, we could add a method to this trait to provide the reason why an expression is incompatible and this could be used to generate documentation.